### ISSUE TITLE
Images don't have the same shape as the real-world collected data

### ISSUE THREAD

**Issue by qgallouedec** [perezjln/gym-lowcostrobot#15]
Real world data have shape (480, 640)

**Comment by perezjln** [comment]
@qgallouedec  - so we should transpose dimension ?

**Comment by qgallouedec** [comment]
No, my point was that currently, observation have shape `(240, 320)` whereas thom's data have shape `(480, 640)`

**Comment by perezjln** [comment]
I see, what could be the solutions ?
 * change resolution ?
 * inform it into the leRobotDataset 
 @Cadene - what could be your recommandation please ?
